modules/update-preparing-ack.adoc

```diff
@@ -24,3 +24,4 @@ Be aware that all responsibility falls on the administrator to ensure that all u
 ----
 $ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.8-kube-1.22-api-removals-in-4.9":"true"}}' --type=merge
 ----
+//This looks right. Is it?
```

Comment:
@bergerhoffer, PTAL?

Reply:
@kalexand-rh yes this is correct as is. Thanks for checking!

Reply:
@bergerhoffer, thank you for confirming!